jumpcloud-ADMU/Powershell/Tests/Public/GUI_JCAdmu.Acceptance.Tests.ps1

```diff
@@ -0,0 +1,621 @@
+Describe "GUI Parameters Acceptance Tests" -Tag "Migration Parameters" {
```

Comment:
Thinking out loud. How much work would it have been to duplicate the tests or add a new case where we test both start-migration and the exe in the same file. If we have to go edit one of the Start-Migration tests we now have to edit these GUI_JCADMU tests in parallel. 

It's duplicated work. As a thought exercise can you think of any scenario where we could have avoided this? It's not my favorite change but I know you had to put a lot of testing time into validating these changes? 

If given more time would you change how this is done?